rust/noosphere-core/src/helpers.rs

```diff
@@ -0,0 +1,48 @@
+// TODO: Figure out how to share this code across crates
```

Comment:
Is this still relevant? `noosphere-sphere` seems to be using it after moving to core

Reply:
Ah, good catch. Yah, this is TODONE.